connect.markdown

```diff
@@ -0,0 +1,23 @@
+---
+layout: fullwidth
+primary_title: Connect
+title: Connect with the Community
+---
+
+## Connect with the Community
+
+---
+
+There are several ways to connect with the OpenSearch community.
+
+* OpenSearch has [active forums](https://discuss.opendistrocommunity.dev/), which are the fastest way to get help using, troubleshooting, and developing with OpenSearch. As a bonus, questions and discussions on the forums are easy for others to learn from in the future. 
```

Comment:
Let's format these as paragraphs without bullets. We may even want to add headings.